overlay/lib.cpp

```diff
@@ -442,6 +442,48 @@ extern "C" __declspec(dllexport) unsigned int __cdecl GetOverlayMagicVersion() {
 	return OVERLAY_MAGIC_NUMBER;
 }
 
+// Via d3d9.cpp
+extern "C" __declspec(dllexport) void __cdecl PrepareD3D9();
```

Comment:
I thought about removing them, but I had second thoughts.

My main reason for not doing it is that there might be programs out there (anti-cheat, other overlays, what have you) that are using the presence of those symbols to detect the Mumble overlay.

For the same reason, I plan to keep the 32-bit Mumble overlay named mumble_ol.dll/mumble_ol.exe. The 64-bit version can then be mumble_ol_x64.dll/mumble_ol_x64.exe or something along those lines.

I think it's better to play it safe.  It's not really doing any harm, and we can always re-visit later.